justfile

```diff
@@ -41,6 +41,12 @@ build_release *ARGS:
 example *ARGS:
   cargo run --profile=release-lto --package hotshot-examples --no-default-features --example {{ARGS}}
 
+example_gpuvid *ARGS:
```

Comment:
Aha!

Reply:
How about rename this to `example_gpuvid_validator` and the next one to `example_gpuvid_leader`, so that users don't need to care about what exact features are enabled, maybe more intuitive?

Reply:
But the GPU is not enabled here, right? Users could get confused.